deployments/warp-deployments.yaml

```diff
@@ -4,39 +4,27 @@
 ---
 description: Hyperlane Warp Route artifacts
 timestamp: 2023-09-23T16:00:00.000Z
+deployer: Abacus Works (Hyperlane)
```

Comment:
The deployer and timestamp are under the `data` list because this file was intended to contain *all* warp deployments from anyone.